TROUBLESHOOTING.md

```diff
@@ -119,13 +119,24 @@ It will start profiling when you start the application, then record the profilin
 Some example usages are:
 
 ```shell script
+PATH_TO_ASYNC_PROFILER=...
+
 # profile CPU startup
-java -agentpath:/path/to/async-profiler/build/libasyncProfiler.so=start,event=cpu,file=startup-cpu-profile.html,interval=1000000,simple\
-    -jar my-application.jar
+java -agentpath:${PATH_TO_ASYNC_PROFILER}/lib/libasyncProfiler.so=start,event=cpu,file=startup-cpu-profile.html,interval=1000000,simple\
+    -jar target/quarkus-app/quarkus-run.jar
 
 # profile allocation startup
-java -agentpath:/path/to/async-profiler/build/libasyncProfiler.so=start,event=alloc,file=/tmp/startup-alloc-profile.html,interval=1000000,simple\
-    -jar my-application.jar
+java -agentpath:${PATH_TO_ASYNC_PROFILER}/lib/libasyncProfiler.so=start,alloc=1,total,event=alloc,file=startup-alloc-profile.jfr -XX:+UnlockExperimentalVMOptions -XX:+UseEpsilonGC -XX:-UseTLAB -Xmx1G -Xms1G -XX:+AlwaysPreTouch -jar target/quarkus-app/quarkus-run.jar
```

Comment:
We can add the reference to this comment from Andrei P. at https://github.com/async-profiler/async-profiler/issues/583#issuecomment-1112757814

Reply:
I would rather keep it simple and not point to comments that will make things more complicated for people not familiar with this.
We just want a command that allows to troubleshoot for people who don't know how to do it.